skills/plan/references/visualize.md

```
@@ -0,0 +1,85 @@
+# VISUALIZE Mode
+
+**Input:** Specs in `.claude/specs/<slug>/`.
+**Output:** Validated Mermaid diagrams at `.claude/specs/<slug>/diagrams.md`.
+
+### Step 1: Find and read the spec
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)